src/components/menu-list/menu-list.stories.js

```diff
@@ -1,78 +1,68 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import { text, boolean } from '@storybook/addon-knobs';
-import { dlsThemeSelector, classicThemeSelector } from '../../../.storybook/theme-selectors';
-import notes from './documentation';
 import { MenuList, MenuListItem } from './menu-list';
-import getDocGenInfo from '../../utils/helpers/docgen-info';
-import docgenInfo from './docgenInfo.json';
 
-MenuList.__docgenInfo = getDocGenInfo(
-  docgenInfo,
-  /menu-list\.js(?!spec)/
-);
-
-MenuListItem.__docgenInfo = getDocGenInfo(
-  docgenInfo,
-  /menu-list-item\.js(?!spec)/
-);
-
-function makeStory(name, themeSelector, disableChromatic = false) {
-  const component = () => {
-    const title = text('title', '');
-    const collapsible = title ? boolean('collapsible', true) : undefined;
-    const filterPlaceholder = text('filterPlaceholder', '');
-    const initiallyOpen = boolean('initiallyOpen', true);
-    const filter = boolean('filter', true);
-
-    return (
-      <MenuList
-        title={ title }
-        collapsible={ collapsible }
-        filterPlaceholder={ filterPlaceholder }
-        initiallyOpen={ initiallyOpen }
-      >
-        <MenuListItem>
-          Menu Item One
-        </MenuListItem>
-        <MenuListItem>
-          <MenuList
-            title='Menu Item Two'
-            filter={ filter }
-          >
-            <MenuListItem name='First Sub Item'>
-              First Sub Item
-            </MenuListItem>
-            <MenuListItem name='Second Sub Item'>
-              Second Sub Item
-            </MenuListItem>
-            <MenuListItem name='Third Sub Item'>
-              Third Sub Item
-            </MenuListItem>
-          </MenuList>
-        </MenuListItem>
-        <MenuListItem>
-          Menu Item Three
-        </MenuListItem>
-        <MenuListItem>
-          Menu Item Four
-        </MenuListItem>
-      </MenuList>
-    );
-  };
-
-  const metadata = {
-    themeSelector,
-    notes: { markdown: notes },
+export default {
+  title: 'MenuList/Test',
+  component: MenuList,
+  parameters: {
+    info: {
+      disable: true
+    },
     knobs: { escapeHTML: false },
     chromatic: {
-      disable: disableChromatic
+      disabled: true
     }
-  };
+  }
+};
 
-  return [name, component, metadata];
-}
+export const Basic = () => {
```

Comment:
shouldn't that be lowercase?

Reply:
All the previous ones I've refactored to CSF have been `Basic`. I'm happy enough to change it to lowercase if that is the correct format though.